### DISCUSSION TITLE
Build for Minecraft version 1.20.6

### DISCUSSION THREAD

**Discussion by Terseus** [P3pp3rF1y/SophisticatedBackpacks#1190] [Q&A]
In the [Curseforge page](https://www.curseforge.com/minecraft/mc-mods/sophisticated-backpacks) there isn't a download link for version 1.20.6, I?m running a 1.20.6 server, can I use a different mod version for it (e.g. 1.21)?

If not, is there any possibility to provide a build for 1.20.6?

**Comment by P3pp3rF1y** [comment]
There's no port for version 1.20.6 so you can't use the mod there. But honestly you should just update to 1.21 where you can use a lot more mods including this one
